pkg/services/system_intakes.go

```diff
@@ -182,7 +167,7 @@ func NewArchiveSystemIntake(
 		}
 
 		// Do note send email if intake was in a draft state (not submitted)
-		if initialStatus != models.SystemIntakeStatusINTAKEDRAFT {
+		if intake.SubmittedAt != nil {
```

Comment:
double check

Reply:
needs to be checked as we are changing conditional

Reply:
This looks right, but I'm not sure if we're able to even "withdraw" (delete) intakes after submission at all. @downeyn-cms @ClayBenson94 ?

Reply:
Might be technically possible from the API perspective, but the UI won't allow it.

Either way, I think this check's valid. We only want to avoid sending an email if it's the first time they're filling out, and have never submitted before